### PR TITLE
Cosmatic Drip 2 ~ Colours of Cargo

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
Part 2 of the Cosmatic Drip PR chain.

This PR focuses entirely on Cargo and Salvage, with a potential template for other departments in the form of the Departmental bandana. This PR also includes a near-full recolour of Cargo's "piss yellow" for a more pleasing yellow-brown colour.

## Why / Balance
Cargo drip! People suggested I do cargo, as well as voiced discontent with the current colours in cargo. This PR will leave most of the code alone regarding lockers and the cargodrobe to prevent merge conflicts with upstream.

## Technical details
Uses upstream path for retextures, as well as upstream's fill for cargodrobe and lockers to prevent conflicts.

## Media
image
image
imageimage
image
image

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

- add: Added Mechanic gloves, a pair of reinforced black gloves to protect from crushes, steel-toed boots, for added on-site safety, and a new form of bandana for departmental use.
- add: Cargo technicians have received two variants of the jumpsuit and jumpskirt. Find 3 of each in the cargodrobe.
- add: Cargodrobe now comes with 3 pairs of mechanics gloves and department bandanas, as well as two steel-toed boots.
- add: QM has received a new loadout set, the Foreman.
- add: Salvage Techs have received a new jumpsuit
- removed: Cargo technicians have lost short privileges
- tweak: Cargo has been recoloured!

### CODE REVIEW

**Comment by KittenColony** [comment]
for the record, the colours I chose were taken from this [Upstream PR](https://github.com/space-wizards/space-station-14/pull/27557).

**Comment by KittenColony** [comment]
professional coder

i forgot a ,

**Comment by Savsj** [comment]
Youre my hero

**Comment by KittenColony** [comment]
de-drafting in order to do a full check, I've been told there are other ways but I don't have the brainpower to find them

**Comment by KittenColony** [comment]
error involves the new bandana, which will be a pain in my *ass* for a while.

**Comment by dffdff2423** [comment]
I will review it once the tests are passing.

**Comment by KittenColony** [comment]
honestly might change the pulled down to a neck item, so you can smoke with it down...

**Comment by dffdff2423** [comment]
Please upload some media for this.

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml [hunk 1]
```diff
@@ -4,16 +4,21 @@
...
     ClothingHeadHatCargosoft: 3
     ClothingHeadBandBrown: 3
+    ClothingMaskBandCargo : 3 # CD
     ClothingHeadsetCargo: 3
     ClothingOuterWinterCargo: 2
     ClothingOuterWinterMiner: 2
     ClothingOuterCoatCargoBomber: 2 # CD
     ClothingNeckScarfStripedBrown: 3
     ClothingShoesBootsWinterCargo: 2
+    ClothingShoesBootsSteel: 2 # CD
```
Comment (dffdff2423) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml:24]: Groups all of the CD items together to reduce merge conflicts.

File: Resources/Prototypes/_CD/Entities/Clothing/Hands/gloves.yml [hunk 2]
```diff
@@ -10,4 +10,23 @@
     sprite: _CD/Clothing/Hands/Gloves/etu_gauntlet.rsi
   - type: Fiber
     fiberMaterial: fibers-nitrile
+  - type: FingerprintMask
+
+- type: entity
+  parent: ClothingHandsBase
+  id: ClothingHandsGlovesMechanic
+  name: mechanics gloves
```
Comment (dffdff2423) [inline Resources/Prototypes/_CD/Entities/Clothing/Hands/gloves.yml:18]: Should be "mechanic's gloves" or "mechanic gloves"

File: Resources/Textures/Clothing/Uniforms/Jumpskirt/qmturtleskirt.rsi/meta.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Derived from beestation qmturtleneck sprite from beestation at commit https://github.com/BeeStation/BeeStation-Hornet/commit/a11e7468ca552cfc97c9164c69ccdafff09e794c and modified by Emisse for Space Station 14",
+  "copyright": "Colour insperation from Wizden's QM Turtle, Sprites by Github KittenColony / Discord kittencolony (297865728374210561)",
```
Comment (dffdff2423) [inline Resources/Textures/Clothing/Uniforms/Jumpskirt/qmturtleskirt.rsi/meta.json:4]: Just checking that you erased all of the wizden sprites if you are doing this?

**Review by dffdff2423 — CHANGES_REQUESTED** [review]
I will finish reviewing this later. This is what I found so far.

**Comment by dffdff2423** [comment]
Discord discussion starts here and continues on for a bit: https://discord.com/channels/1163352113162768436/1163713936231764059/1282425058148614226

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml (hunk 1 above)
Comment (KittenColony) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml:24]: It's like this to preserve the order they're shown in the cargodrobe though, not sure what you mean

File: Resources/Textures/Clothing/Uniforms/Jumpskirt/qmturtleskirt.rsi/meta.json (hunk 3 above)
Comment (KittenColony) [reply Resources/Textures/Clothing/Uniforms/Jumpskirt/qmturtleskirt.rsi/meta.json:4]: QM turtleneck uses my own top template. The others were hueshifts and it feels wrong to change the copyright after just that

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml (hunk 1 above)
Comment (dffdff2423) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml:24]: Oh, then that is fine. By merge conflicts I am referring to the conflicts created when Lank merges upstream. Having all of the CD changes in one block helps with that.

**Comment by KittenColony** [comment]
going to work on a few more things to put in the PR so I will say, don't merge just yet

**Comment by KittenColony** [comment]
assuming all this passes, ready for re-review / merging

**Comment by KittenColony** [comment]
ready to go

**Comment by dffdff2423** [comment]
In the future, just make another PR when you want to add stuff. This PR is massive. I also would have preferred that the additions and replacements be in separate PRs.

**Review by dffdff2423 — DISMISSED** [review]
We discussed this PR internally and decided that the RSI changes should be namespaced.

We also want to know why the vox sprite is so large.

**Comment by dffdff2423** [comment]
It is 8 KB for some reason

**Comment by KittenColony** [comment]
> It is 8 KB for some reason

literally have zero idea why, let me try something

**Comment by KittenColony** [comment]
oh godfucking damn it

**Comment by KittenColony** [comment]
OH WELL I GUESS ITS JUST STUCK THERE

**Review by Nairodian — APPROVED** [review]
Sprite wise, looks real good on paper, no outstanding comments from me.

**Review by dffdff2423 — APPROVED** [review]
Looks fine. Please try to make smaller PRs in the future. This is basically the same size as the last one.

**Comment by Everturning** [comment]
this is very much not good, look at the images again, these sprites are noticably offset from the urist. you can see it the most on the shoulder/arm areas on all of the sprites

**Comment by KittenColony** [comment]
> this is very much not good, look at the images again, these sprites are noticably offset from the urist. you can see it the most on the shoulder/arm areas on all of the sprites they look good but they are offset

image

not my problem not my fault

**Pull request merged**